examples/waspello/src/cards/MainPage.jsx

```diff
@@ -248,7 +248,7 @@ const List = ({ list, index, cards }) => {
           {...provided.draggableProps}
           {...provided.dragHandleProps}
         >
-          <div className='list'>
+          <div className={`list ${snapshot.isDragging && 'list--dragging'}`}>
```

Comment:
I'd implement a helper function for this e.g. 

```ts
function getListClassName(snapshot) {
  const classes = ['list']
  if (snapshot.isDragging) {
    classes.push('list--dragging')
  }
  if (snapshot.isDropAnimating) {
    classes.push('list--drop-animating')
  }
  return classes.join(' ')
}
```

This way:
1. There is no space after `list` if `list--dragging` is not present. 
2. It's easier to compose multiple classes

You'll notice I used `isDropAnimating` as well, this enables us to have a smooth start:
```
(no rotation) -> (rotation)
```
and a smooth end:
```
(rotation) -> (no rotation)
```

https://github.com/user-attachments/assets/0e22888d-7fe2-40a0-bf37-2b652dc6e530


https://github.com/user-attachments/assets/a9c4fab4-6264-48e0-ba29-1d57f52d5234

Reply:
I just noticed that `classnames` npm package is included and used in other places. Should have used that 🤦
I'll update the code.